### DISCUSSION TITLE
PT-BR translation

### DISCUSSION THREAD

**Discussion by doggrush** [aclap-dev/video-downloadhelper#937] [Language & Translation]
Here are the last 6 strings that needed to be translated to portuguese brazillian.
-> **[pt_BR.txt](https://github.com/user-attachments/files/16007557/pt_BR.txt)** <-

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
